configure.in

```diff
@@ -386,11 +397,26 @@ case "${host_os}" in
         if [[ "$have_jack" = "yes" ] && [ "$with_jack" != "no" ]] ; then
            DLL_LIBS="$DLL_LIBS $JACK_LIBS"
            CFLAGS="$CFLAGS $JACK_CFLAGS"
-           OTHER_OBJS="$OTHER_OBJS src/hostapi/jack/pa_jack.o src/common/pa_ringbuffer.o"
+           OTHER_OBJS="$OTHER_OBJS src/hostapi/jack/pa_jack.o"
            INCLUDES="$INCLUDES pa_jack.h"
            AC_DEFINE(PA_USE_JACK,1)
         fi
 
+        if [[ "$have_pulse" = "yes" ] || [ "$have_jack" = "yes" ]] ; then
+           OTHER_OBJS="$OTHER_OBJS src/common/pa_ringbuffer.o"
+           INCLUDES="$INCLUDES pa_linux_pulseaudio.h"
```

Comment:
This will add `pa_linux_pulsaudio.h` even if only `$have_jack` is yes, which is not what you want.

Reply:
This PR is merged now, but I'd still like to get clarification about whether this has the intended behavior please.

Reply:
Seems to be copy paste.. I'll make PR to solve this. I think it should be something like `"$have_pulse" = "yes" ] && [ "$with_pulse" != "no"` or just merge it to one to rule them all if. I have to admit I use CMake so this was not tested very deeply.

Reply:
Now there is pull request with small fix to make it work. Jack and Pulseaudio needs ringbuffer so it's just merged together.